runtime/lua/vim/filetype.lua

```diff
@@ -2489,7 +2489,10 @@ function M.match(args)
 
   -- Finally, check file contents
   if contents or bufnr then
-    contents = contents or M.getlines(bufnr)
+    if contents == nil then
+      contents = M.getlines(bufnr, 1, 100)
+      table.insert(contents, M.getlines(bufnr, -1))
```

Comment:
Feel free to completely ignore this comment.

Tiny nitpick, but I pretty much always avoid `table.insert`:

```suggestion
     contents[#contents+1] = M.getlines(bufnr, -1)
```
 As it requires less 1 table access + a function call.

Obviously in this context, any perf differences are going to be absolutely negligible, however both are just as readable, so I just always opt for the latter for general consistency. There are very few cases where `table.insert` is actually worth using.

Reply:
Haha, you did this suggestion before I could even write this comment.

Reply:
Yeah. I checked, but I don't think we're consistent in using one or the other.

I like `table.insert` because it's more readable, but it makes the language server throw a hissy fit about something so I ended up changing it after all.